filesystem/rootfs/docker/scripts/clean-docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+systemctl stop docker
+rm -rf /etc/docker/daemon.json
+rm -rf /lib/systemd/system/docker.service
+rm -rf /usr/lib/systemd/system/docker.service
+systemctl daemon-reload
+
+rm -f /usr/bin/containerd
+rm -f /usr/bin/containerd-shim
+rm -f /usr/bin/ctr
+rm -f /usr/bin/docker
+rm -f /usr/bin/docker-init
+rm -f /usr/bin/docker-proxy
+rm -f /usr/bin/dockerd
+rm -f /usr/bin/runc
```

Comment:
we need to find out all of the binaries install by sealer and delete it all. 
i'm not sure  `/usr/bin/containerd-shim-runc-v2` should be list in here ?

Reply:
at newest image `/usr/bin/containerd-shim-runc-v2`  already not exist